app/assets/stylesheets/v3/layout_components/footer.scss

```diff
@@ -52,35 +52,187 @@ body {
+      max-width: 992px;
+    }
+    @include breakpoint(xlarge only) {
+      max-width: 1334px;
+    }
+  
+    @include breakpoint(xxlarge up) {
+      max-width: 1440px;
+    }
   }
 
   &__col {
-    @include medium-6;
     @include large-3;
+    display: none;
+
+    @include breakpoint(large up) {
+      max-width: 992px;
+      display: block;
+    }
+  }
+
+  &__col-mobile {
+
+
+    @include breakpoint(large up) {
+      max-width: 992px;
+      display: none;
+    }
+  }
+
+  &__col-right {
+    @include large-2;
   }
 
   &__col-header {
+    font-size: $base-font-size*1.5;
+    font-weight: 900;
     color: $white;
-    font-weight: bold;
+    padding-bottom: $base-padding;
+    line-height: 1.2;
+    position: relative;
+
+    &--small {
+      font-size: $base-font-size*0.875;
+      font-weight: 900;
+      color: $white;
+      padding-bottom: $base-padding;
+      line-height: 1.2;
+    }
   }
 
-  &__list {
-    list-style-type: none;
+  //This is to line up the social media icons with the text
+  &__col:last-child &__col-header { 
+    padding-bottom: $base-padding*1.5;
   }
 
-  &__link {
-    color: $white;
+  &__col-social {
+    padding-bottom: $base-padding*2;
+
+    @include breakpoint(large up) {
+      padding-bottom: $base-padding*8;
+    }
+
+    img {
+      width: 40px;
+
+      @include breakpoint(large only) {
+        width: 30px;
+      }
+    }
+  }
+
+  &__policy {
+    @include grid-row;
+    margin-top: $base-margin;
+    padding-top: $base-padding*2;
+    border-top: 1px solid $white;
+
+    @include breakpoint(medium only) {
+      max-width: 704px;
+    }
+  
+    @include breakpoint(large only) {
+      max-width: 992px;
+    }
+
+    @include breakpoint(xlarge only) {
+      max-width: 1334px;
+    }
+  
+    @include breakpoint(xxlarge up) {
+      max-width: 1440px;
+    }
+
+    .footer-link {
+      color: $white;
+      font-size: $base-font-size;
+    }
+
+    .footer-link:hover {
+      color: $white;
+    }
+  }
+
+  .link-list {
+    display: none;
+
+    @include breakpoint(large up) {
+      display: inline;
+    }
+  }
+}
+
+.policy-text {
+  text-align: center;
+  padding-bottom: $base-padding;
+
+  @include breakpoint(large up) {
+    text-align: right;
+    padding-bottom: 0;
+  }
+}
+
+.policy-ama {
+  text-align: center;
+  padding-top: $base-padding;
+
+  @include breakpoint(large up) {
+    text-align: left;
+    padding-top: 0;
   }
 }
 
-.racetrack {
-  background: $brand-blue-light image-url("footer-racetrack-big.png") no-repeat center $base-padding*3;
-  height: 160px;
+.affiliate-link {
+  text-align: center;
+  color: $white;
+  font-size: $base-font-size;
+
+  p {
+    line-height: 1rem!important;
+    margin-top: $base-margin*0.5;
+  }
+
+}
+
+.footer-copyright {
+  text-align: center;
+
+  @include breakpoint(large up) {
+    text-align: right;
+  }
+}
+
+.is-accordion-submenu-parent > a:after {
+  display: block;
+  font-family: "Font Awesome 5 Free";
+  content: "\f078";
```

Comment:
if we put this in as content: fa-content($chevron-down);

this makes it dynamic, it pulls it from the mapping in the font awesome gem, and if they change the codes in the future this should still map it to the correct icon. (Zoie taught me this if you want more info she can explain it well)

Reply:
Thanks! I'll look into it!